src/Commands/Files/MoveFile.cs

```diff
@@ -83,8 +83,9 @@ protected override void ExecuteCmdlet()
                         var folderServerRelativePath = folder.EnsureProperty(f => f.ServerRelativePath);
                         isFolder = folderServerRelativePath.DecodedUrl == ResourcePath.FromDecodedUrl(TargetUrl).DecodedUrl;
                     }
-                    catch
+                    catch (Exception ex)
                     {
+                        WriteWarning(ex.Message);
```

Comment:
I understand why you want to visualize this situation, not sure yet if warning is the best method for this as it may break automated builds that are using this today. Verbose might be a safer option. @gautamdsheth What's your take on this?

Reply:
Yeah, `WriteVerbose` is the safer option for this exception, we don't want to break scripts.

Reply:
All right thank you for you feedback - I'll change it and update the branch